iOS/DuckDuckGo/NetworkProtectionStatusView.swift

```diff
@@ -354,35 +354,32 @@ struct NetworkProtectionStatusView: View {
     @available(iOS 18.0, *)
     @ViewBuilder
     private func geoswitchingTipView() -> some View {
-        if statusModel.canShowTips {
-            TipView(tipsModel.geoswitchingTip)
-                .removeGroupedListStyleInsets()
-                .tipCornerRadius(0)
-                .tipBackground(Color(designSystemColor: .surface))
-                .onAppear {
-                    tipsModel.handleGeoswitchingTipShown()
-                }
-                .task {
-                    var previousStatus = tipsModel.geoswitchingTip.status
+        TipView(tipsModel.geoswitchingTip)
```

Comment:
This is just removing the containing `if`, which results in an indentation change for the entire block.